tests/src/test/java/alluxio/worker/DataServerIntegrationTest.java

```diff
@@ -106,6 +106,7 @@ public final void before() throws Exception {
   @After
   public final void after() throws Exception {
     mBlockMasterClient.close();
+    BlockStoreContext.get().releaseWorkerClient(mBlockWorkerClient);
```

Comment:
I don't see we acquire block worker client in that test. I was looking at BlockServiceHandlerIntegrationTest.java